src/seedsigner/views/seed_views.py

```diff
@@ -223,6 +254,8 @@ def run(self):
             title=_("Seed Word #{}").format(self.cur_word_index + 1),  # Human-readable 1-indexing!
             initial_letters=list(self.cur_word) if self.cur_word else ["a"],
             wordlist=Seed.get_wordlist(wordlist_language_code=self.settings.get_value(SettingsConstants.SETTING__WORDLIST_LANGUAGE)),
+            #TODO: add a standardized way to get the possible alphabet
+            possible_alphabet=_("abcdefghijklmnopqrstuvwxyz"),
```

Comment:
This should NOT be wrapped for translation.

If a non-english wordlist is selected, we need to be explicit in the code what the expected character set is.

Also, per the TODO comment, I would have the `possible_alphabet` defined for each language in a constant either in `SettingsConstants` (hmm, it's not really a Setting though) or in `Seed` where the wordlist lookup is provided.

That being said, I don't love that the wordlist lookup is in `Seed`. Perhaps a `util` module within `helpers.bip39`?